### ISSUE TITLE
Learnable Multi-level Frequency Decomposition and Hierarchical Attention Mechanism for Generalized Face Presentation Attack Detection

### ISSUE THREAD

**Issue by jrzmnt** [unico-labs/ml-reading-club#73]
### Learnable Multi-level Frequency Decomposition and Hierarchical Attention Mechanism for Generalized Face Presentation Attack Detection

_With the increased deployment of face recognition systems in our daily lives, face presentation attack detection (PAD) is attracting much attention and playing a key role in securing face recognition systems. Despite the great performance achieved by the hand-crafted and deep-learning based methods in intra-dataset evaluations, the performance drops when dealing with unseen scenarios. In this work, we propose a dual-stream convolution neural networks (CNNs) framework. One stream adapts four learnable frequency filters to learn features in the frequency domain, which are less influenced by variations in sensors/illuminations. The other stream leverages the RGB images to complement the features of the frequency domain. Moreover, we propose a hierarchical attention module integration to join the information from the two streams at different stages by considering the nature of deep features in different layers of the CNN. The proposed method is evaluated in the intra-dataset and cross-dataset setups, and the results demonstrate that our proposed approach enhances the generalizability in most experimental setups in comparison to state-of-the-art, including the methods designed explicitly for domain adaption/shift problems. We successfully prove the design of our proposed PAD solution in a stepwise ablation study that involves our proposed learnable frequency decomposition, our hierarchical attention module design, and the used loss function. Training codes and pre-trained models are publicly released._

[Link para o Paper](https://openaccess.thecvf.com/content/WACV2022/papers/Fang_Learnable_Multi-Level_Frequency_Decomposition_and_Hierarchical_Attention_Mechanism_for_Generalized_WACV_2022_paper.pdf)

[Link para o Github](https://github.com/meilfang/LMFD-PAD)

![workflow](https://user-images.githubusercontent.com/5489918/148558892-834f80d0-d047-4cea-8fd3-db2569a1cadd.png)

**Comment by jrzmnt** [comment]
THIS
